src/foam/nanos/approval/ApprovalRequest.js

```diff
@@ -570,13 +573,28 @@
           this.notify(this.SUCCESS_REJECTED, '', this.LogLevel.INFO, true);
 
           if ( this.stack.top.length > 0 ) {
-            this.stack.back();
+            if ( ! stackBackIgnore )
+              this.stack.back();
+            // else
```

Comment:
we need to update table as otherwise we will not be able to observe changes immediately.

we have `this.data.sub('on', function()` in UnstyledTableView line 410 but that's not triggered (not sure if it works as expected)